vue/wolox-books/src/views/Register.vue

```diff
@@ -0,0 +1,162 @@
+<template lang='pug'>
+  .register-container
+    img.wolox-icon(alt='Vue logo' src='../assets/wolox_logo.svg')
+    form.form-container(@submit.prevent="onSubmit")
+      p.title-form
+        |{{labels.title}}
+      .input-text-container(v-for='(field, index) in fields')
+        label.input-text-label(:for='field.name')
+          |{{ field.label }}
+        input.input-text-content(:id='field.name' v-model='user[field.name]' )
+      button.base-form-button
+        |{{ labels.signUp }}
+    .container-button
+      button.base-form-button.login-button
+        |{{ labels.signIn }}
+</template>
+
+<script>
+
+const fieldsArray = [
+  {
+    name: 'fisrtName',
+    label: 'First name'
+  },
+  {
+    name: 'lastName',
+    label: 'Last name'
+  },
+  {
+    name: 'email',
+    label: 'Email'
+  },
+  {
+    name: 'password',
+    label: 'Password'
+  }
+]
+
+const labels = {
```

Comment:
you really need this as a constant? This texts will be used in i18n afterwards

Reply:
Sorry i forget put binding key with index. :key='index'